src/promptflow/promptflow/_sdk/_service/apis/run.py

```diff
@@ -220,3 +220,12 @@ class RestoreRun(Resource):
     def get(self, name: str):
         run = get_client_from_request().runs.restore(name=name)
         return jsonify(run._to_dict())
+
+
+@api.route("/<string:name>/delete")
```

Comment:
I understand this should be same path as GET one?

Reply:
Yes